Source/Core/Core/PowerPC/JitArm64/JitArm64_BackPatch.cpp

```diff
@@ -369,17 +377,6 @@ bool JitArm64::HandleFault(uintptr_t access_address, SContext* ctx)
 		return false;
 	}
 
-	ARM64Reg reg = INVALID_REG;
-	u32 flags = 0;
-
-	if (!DisasmLoadStore((const u8*)ctx->CTX_PC, &flags, &reg))
```

Comment:
This is the last call to DisasmLoadStore, so shall we also remove this function completely?